### ISSUE TITLE
date range for headways and trip times

### ISSUE THREAD

**Issue by mjrerle** [trynmaps/metrics-mvp#55]
ability to have more than one date and add time ranges

**Comment by exxonvaldez** [comment]
I think we still need this issue open for arrivals/trip time data over a date range as opposed to a single date.  I’ll reopen and edit out the time range part.

**Comment by hathix** [comment]
good call. hopefully borrowing from uber movement's UI will help.

**Comment by TheLordHighExecutioner** [comment]
> good call. hopefully borrowing from uber movement's UI will help.

Do you me using Uber's UI library, or use material-ui to create a similar user interface?

**Comment by exxonvaldez** [comment]
> > good call. hopefully borrowing from uber movement's UI will help.
> 
> Do you mean using Uber's UI library, or use material-ui to create a similar user interface?

This means using Material-UI to create a similar interface.  As far as I can tell, the Uber movement date picker UI is propietary, although I'm not totally sure.  We need the front end to have more date options (although when the back end can actually support them is an open question).  

We want the date picker to support having a "from" date and a "to" date, although these might be fixed buckets (e.g., specific quarters like Q3 2018, or last 30 days) depending on how we implement date ranges across all the OpenTransit features.  Currently every feature we have is designed to work with just one day's data.

We also want widgets to allow selection of days of the week like in Uber Movement.  Again, what's actually supported may be a subset, like initally perhaps a choice of all days / weekdays / weekends and then later choosing one day of the week and then arbitrary days of the week.

**Comment by TheLordHighExecutioner** [comment]
Got it. This seems like it needs a small design doc?

>  As far as I can tell, the Uber movement date picker UI is propietary, although I'm not totally sure.

It appears to be open source: https://baseweb.design/components/datepicker/

**Comment by hathix** [comment]
This still needs to be done. Looking for an eager new person (or anyone, really) to make this happen. The app UI has changed significantly since this issue was stated -- it now looks like the below.
![Screen Shot 2019-09-11 at 4 59 55 PM](https://user-images.githubusercontent.com/728084/64743644-a1c93d00-d4b5-11e9-89e4-181ee74826c4.png)

**Comment by exxonvaldez** [comment]
Just a note that this is blocked by #177 , so UI changes could be made first but would have to be kept hidden (via a feature flag).

**Comment by hathix** [comment]
This is a duplicate of #301 and sorta #321. I'm going to close this because it's duplicative, but it's good to reference it.
